### DISCUSSION TITLE
Does it have webhook support?

### DISCUSSION THREAD

**Discussion by ilovefreesw** [evershopcommerce/evershop#513] [Q&A] [answered]
I was looking at API docs and didn't see any webhook support? Does it have webhook support to subscribe to events such as product CRUD operations, new orders, abandoned cart, etc?

**Comment by treoden** [comment, marked as answer]
Hi @ilovefreesw ,

EverShop supports event/subscriber. You can create your own subscriber to subscriber to particular event like product created, order created.
For more information you can check here: https://evershop.io/docs/development/knowledge-base/events-and-subscribers
